erdpy/config.py

```diff
@@ -51,14 +51,48 @@ def get_dependency_url(key: str, tag: str, platform: str) -> str:
 
 def get_value(name: str) -> str:
     _guard_valid_name(name)
-    data = read_file()
+    data = get_active()
     return data.get(name, get_defaults()[name])
 
 
 def set_value(name: str, value: Any):
     _guard_valid_name(name)
     data = read_file()
-    data[name] = value
+    data["configurations"][data["active"]][name] = value
```

Comment:
Perhaps use `theDictionary.get(key, fallback)` pattern instead (currently the build on Github Actions fails, somehow related to this).

Reply:
done, looks decent but not really happy, if you have a suggestion for prettier code, I'm open 🥇